build/tools/appveyor-test.bat

```diff
@@ -9,27 +9,27 @@ goto %TOOLSET%
 :msbuild
 PATH=C:\projects\wxwidgets\lib\vc_x64_dll;%PATH%
 .\vc_x64_mswudll\test.exe
-if errorlevel 1 goto :error
+if %errorlevel% NEQ 0 goto :error
```

Comment:
Just out of curiosity, what's the difference between the old and the new versions? AFAIK they should do the same thing, don't they?

Reply:
When I got a segfault, it returned `-1073741819`, not `1`. Currently the segfault is not correctly detected, see for example [this test](https://ci.appveyor.com/project/wxWidgets/wxwidgets/build/3387/job/j5a4kaet7bch95nd#L1771). It just stops (no finished message or summary) and continues with the next test. I hope this will detect it.

Reply:
[this test](https://ci.appveyor.com/project/wxWidgets/wxwidgets/build/3408/job/61opifr6q26tjf13#L1758) shows the new version does detect the error.